requirements.in

```diff
@@ -4,6 +4,7 @@ flake8-commas
 isort
 multilint
 patchy
+pathlib2
```

Comment:
Why?

Reply:
this package is needed for python 3.5. I could add a condition if you want

Reply:
Hmm I'd prefer not to move to a test grid with Python 3.5 etc. at this time. Just focus the PR on Django 2.2 support. I have a template I keep all my OSS projects conforming to, changing one of them away from that creates unnecessary burden on me.

Reply:
ok, no pb :) but the readme should be changed to remove the python 3.4-3.5 support, no ?

Reply:
I support Python 3.4+, just don't run the tests on it on Travis. It's a pure Python package, I know the differences from 3.5+ and just don't use them. If there's a compat issue I'll move to grid testing all the Python versions, but I'll do it on all repos at once.

Reply:
done :)